node/cmd/guardiand/node.go

```diff
@@ -323,6 +325,8 @@ func init() {
 	optimismCtcContractAddress = NodeCmd.Flags().String("optimismCtcContractAddress", "", "Optimism CTC contract address")
 
 	logLevel = NodeCmd.Flags().String("logLevel", "info", "Logging level (debug, info, warn, error, dpanic, panic, fatal)")
+	logPublicRPC = NodeCmd.Flags().String("logPublicRPC", "full", "Logging of public RPC requests (false=no logging, minimal=only log gRPC methods, full=log gRPC method, payload (up to 200 bytes) and user agent (up to 200 bytes))")
```

Comment:
Who is enforcing the 200 byte limit?

Reply:
for the body, it's here: https://github.com/wormhole-foundation/wormhole/blob/c957120b5570db63b1c441a26d60c8750a7dad41/node/pkg/common/grpc.go#L71
for the user-agent, it's indeed missing after I refactored some code. Will add it back in, good catch!